docs/README.md

```diff
@@ -310,13 +320,74 @@ https://codeburst.io/javascript-passing-by-value-vs-reference-explained-in-plain
 
 <br/>
 
-## 5. Timer function
+## 6. Timer function
 
 There are two methods for it:
 
 - setTimeout allows us to run a function once after the interval of time.
 - setInterval allows us to run a function repeatedly, starting after the interval of time, then repeating continuously at that interval.
 
+## 7. Errors
+
+![error](https://i.ibb.co/jD2kG28/e.jpg)
+
+""To err is human..."  Everyone makes a mistake, even the seasoned developer. Hence, it's essential to know about different errors and how to solve them. Errors can be divided into two groups: compile-time errors and run-time errors.
+
+#### A. Compile-time errors
+
+Compile-time errors are a class of errors which prevents the program from compiling:
+
+- syntax error: incorrect keyword, a forgotten symbol
+- calling a non-existing method
+- wrong source code file name
+
+In the case of JavaScript, compile-time errors occur at interpret time. For example, in the below code, a syntax error occurs because it is missing semi-colon ';' at the end of the print() function invocation.
```

Comment:
In js ; are optional. So, I think this couldn't result in an error
Try adding another example